pages/pipelines/permissions.md.erb

```diff
@@ -85,9 +86,10 @@ query FindOrganizationMember {
     }
   }
 }
-```  
+```   
```

Comment:
If you were being really picky you could delete the trailing spaces also.

Reply:
Done. As I AM picky ;)